composite-view/src/test/java/com/illuwatar/compositeview/JavaBeansTest.java

```diff
@@ -0,0 +1,59 @@
+package com.illuwatar.compositeview;
+
+import com.iluwatar.compositeview.ClientPropertiesBean;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.Assert.*;
+
+public class JavaBeansTest {
+    @Test
+    public void testDefaultConstructor() {
+        ClientPropertiesBean newBean = new ClientPropertiesBean();
+        assertEquals("DEFAULT_NAME", newBean.getName());
+        assertTrue(newBean.isBusinessInterest());
+        assertTrue(newBean.isScienceNewsInterest());
+        assertTrue(newBean.isSportsInterest());
+        assertTrue(newBean.isWorldNewsInterest());
+
+    }
+
+    @Test
+    public void testNameGetterSetter() {
```

Comment:
Are tests for generated setter/getters really necessary?

Reply:
I will write some mocking tests for the AppServer and the non-empty constructor, but if I remove these getter/setter tests, I think the sonar cloud analysis will still fail, due to not enough line coverage.